test/e2e/tests/breakpoints-01.test.ts

```diff
@@ -0,0 +1,12 @@
+import config from "../config";
+import { runClassicTest } from "../runTest";
+
+it("Test basic breakpoint functionality.", async () => {
```

Comment:
I think these `classic` tests could be expressed through `jest-in-case` as well which would cut down on the duplication quite a lot. I think you'd still be able to get all of the jest watch utilities as well

https://www.npmjs.com/package/jest-in-case

Reply:
I had considered something similar, but failed to get it to work with the VS Code integration for jest. But most importantly this is the structure I think it will have when we move away from the classic tests.

Reply:
Did it work from the command line? What was breaking in VSCode? I've never run the tests from VSCode, but I'd be happy to help try and get it to work, getting rid of that duplication would be 🤌

Reply:
In the end I decided I could live without VS Code integration, but there's another reason why this approach wouldn't work: jest's `--shard` option, which we need in CI, puts all tests from the same file in the same "shard".
I think we could hack our own stripe/shard implementation and then we could also follow this approach. Not sure if it's worth it though.